geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -103,51 +103,50 @@ object AccumuloDataStoreFactory extends GeoMesaDataStoreInfo {
       "",
       false,
       null,
-      ImmutableMap.of(Parameter.DEPRECATED, true, Parameter.IS_PASSWORD, true))
+      Map(Parameter.DEPRECATED -> true, Parameter.IS_PASSWORD -> true).asJava)
 
   override def canProcess(params: java.util.Map[String, _ <: Serializable]): Boolean = {
-    val hasConnector = ConnectorParam.lookupOpt(params).isDefined
-    def hasConnection = InstanceIdParam.exists(params) && ZookeepersParam.exists(params) && UserParam.exists(params)
-    def hasPassword = PasswordParam.exists(params) && !KeytabPathParam.exists(params)
-    def hasKeytab = !PasswordParam.exists(params) && KeytabPathParam.exists(params) && isKerberosAvailable
-    hasConnector || (hasConnection && (hasPassword || hasKeytab))
+    val hasConnection = InstanceIdParam.exists(params) && ZookeepersParam.exists(params) && UserParam.exists(params)
+    // exactly one of password or keytab
+    hasConnection && (PasswordParam.exists(params) != KeytabPathParam.exists(params))
   }
 
   def buildAccumuloConnector(params: java.util.Map[String, _ <: Serializable]): Connector = {
     if (MockParam.lookup(params)) {
       throw new IllegalArgumentException("Mock Accumulo connections are not supported")
```

Comment:
Should we remove this? 

Did you git grep -l mock:)?

Reply:
i did git grep mock. and removed most references. I think keeping this here is useful in case someone is still trying to use the old param for whatever reason.